arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -0,0 +1,1742 @@
+                },
+                "createdAt": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The time the namespace was created."
+                },
+                "serviceBusEndpoint": {
+                    "externalDocs": {
+                        "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+                    },
+                    "type": "string",
+                    "description": "Endpoint you can use to perform NotificationHub operations."
+                },
+                "subscriptionId": {
+                    "type": "string",
+                    "description": "The Id of the Azure subscription associated with the namespace."
+                },
+                "scaleUnit": {
+                    "type": "string",
+                    "description": "ScaleUnit where the namespace gets created"
+                },
+                "enabled": {
+                    "type": "boolean",
+                    "description": "Whether or not the namespace is currently enabled."
+                },
+                "critical": {
+                    "type": "boolean",
+                    "description": "Whether or not the namespace is set as Critical."
+                },
+                "namespaceType": {
+                    "type": "string",
+                    "description": "Gets or sets the namespace type.",
+                    "enum": [
+                        "Messaging",
+                        "NotificationHub"
+                    ],
+                    "x-ms-enum": {
+                        "name": "NamespaceType",
+                        "modelAsString": false
+                    }
+                }
+            },
+            "description": "Namespace properties."
+        },
+        "NamespaceCreateOrUpdateParameters": {
+            "properties": {
+                "location": {
+                    "type": "string",
+                    "description": "Gets or sets Namespace data center location."
+                },
+                "tags": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    },
+                    "description": "Gets or sets Namespace tags."
+                },
+                "properties": {
+                    "$ref": "#/definitions/NamespaceProperties",
+                    "description": "Gets or sets properties of the Namespace."
+                }
+            },
+            "required": [
+                "location",
+                "properties"
+            ],
+            "description": "Parameters supplied to the CreateOrUpdate Namespace operation."
+        },
+        "NamespaceResource": {
+            "properties": {
+                "id": {
```

Comment:
same comment as before. NamespaceResource must be a child of Resource. Have an allof on the resource. and remove resource properties from this model